### PR TITLE
fix bugs, add log and add correction

### PR DESCRIPTION
**Pull request by MingyueX**

### CODE REVIEW

File: trees/app/src/main/java/com/trees/common/pyi/ImageProcessor.java [hunk 1]
```diff
@@ -57,6 +66,13 @@ public ImageResult processImage(Activity context, ImageRaw raw) {
                 Toast.makeText(context, R.string.noDepthPoints, Toast.LENGTH_LONG).show();
             }
 
+            // log the info
+            Log.i("Miranda", "Log info:" + logInfo);
```
Comment (ameliaholcomb) [inline trees/app/src/main/java/com/trees/common/pyi/ImageProcessor.java]: Could you clean this up to use a standard log string? I know there are other places hanging around where I do this with "Amelia" ... but better to move slowly towards a better code style :)

**Review by ameliaholcomb — APPROVED** [review]

File: trees/app/src/main/java/com/trees/common/pyi/ImageProcessor.java (hunk 1 above)
Comment (MingyueX) [reply trees/app/src/main/java/com/trees/common/pyi/ImageProcessor.java]: Sure, will do!

Comment (MingyueX) [reply trees/app/src/main/java/com/trees/common/pyi/ImageProcessor.java]: I changed the logtag to "IntermediateData", is that what you are suggesting? Please correct me if I've misconstrued. Also, would you like me to modify all occurrences of those non-standard log strings?

**Comment by ameliaholcomb** [comment]
This looks good, thanks! I'm going to merge your pull request.

**Pull request merged**